src/StringHelper.php

```diff
@@ -417,6 +417,9 @@ public static function uppercaseFirstCharacter(string $string, string $encoding
      */
     public static function uppercaseFirstCharacterInEachWord(string $string, string $encoding = 'UTF-8'): string
     {
+        /**
+         * @var array $words We assume that `$string` is valid UTF-8 string, so `preg_split()` never returns `false`.
```

Comment:
Makes sence to mention it in the parameter description above.
For other methods also.

Reply:
Done